docs/source/compute_config/aws_lambda.md

```diff
@@ -52,26 +51,69 @@ python3 -m pip install lithops[aws]
 lithops:
     backend: aws_lambda
 
-aws:
+aws_lambda:
+    execution_role: <EXECUTION_ROLE_ARN>
     region: <REGION_NAME>
```

Comment:
Is the execution_role mandatory in aws_lambda? if yes I would update this .md file and include it in all the parts where you put some lithops config example, to make it clearer. Is region mandatory in all the cases? I think this lithops config example is confusing here. I would remove it and put the config example in the next section, when necessary, with all the necessary parameters.

Reply:
Yes, `execution_role` is mandatory. The user must specify which services can the lambda access to. We could automate this, but the user should have IAM permissions like create role... We can leave like this for now